doc/flame/game.md

```diff
@@ -93,6 +93,20 @@ in order: `onLoad`, `onGameResize` and `onMount`. After that, it goes on to call
 Once the `GameWidget` is removed from the tree, `onRemove` is called, just like when a normal
 component is removed from the component tree.
 
+```{note}
+The `onRemove` can be used to clean up potential memory leaks by doing the following:
+```
+
+```dart
+  @override
+  void onRemove() {
+    removeAll(children);
+    processLifecycleEvents();
+    // Any other code necessary to remove references that will be orphaned when
+    // your game is removed.
```

Comment:
```suggestion
    // Any other code that you want to run when the game is removed.
```
I think this can be simplified to this, because these references should already be handled by the GC if you just make sure that you don't keep a reference to the game class itself.